### ISSUE TITLE
new project for icetray-independent support scripts (Trac #1010)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1004]
I'd like a project where I can create "officially sanctioned" support scripts.  They should not be tied to an icetray release version, so trunk can be used in older releases.

Example uses:
- how to get all the files from run/dataset X
- where is the gcd file for run/dataset X
and probably more.

Not sure about naming.  I guess I leave that up to olivas?

Migrated from https://code.icecube.wisc.edu/ticket/1010
```json
{
    "status": "closed", 
    "changetime": "2015-06-16T22:12:23", 
    "description": "I'd like a project where I can create \"officially sanctioned\" support scripts.  They should not be tied to an icetray release version, so trunk can be used in older releases.\n\nExample uses:\n- how to get all the files from run/dataset X\n- where is the gcd file for run/dataset X\nand probably more.\n\nNot sure about naming.  I guess I leave that up to olivas?", 
    "reporter": "david.schultz", 
    "cc": "nega", 
    "resolution": "fixed", 
    "_ts": "1434492743892459", 
    "component": "combo reconstruction", 
    "summary": "new project for icetray-independent support scripts", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-06-04T16:04:15", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150604T16:42:11: olivas commented:

> I'm going to add this to the bootcamp agenda.  Being able to find data and GCD files is something the group will collectively want to know how to do.

**Comment by icecube-trac** [comment]
Trac update at 20150604T16:42:34: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150616T22:12:23: 
* nega changed resolution from "" to "fixed"
* nega commented:

> In changeset 133937:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="133937"
> create a project for support/data-search scripts
> 
> closes #1004
> ```


* nega changed status from "accepted" to "closed"
